fbw-common/src/systems/instruments/src/OANC/style-data.ts

```diff
@@ -1,115 +1,164 @@
 // Copyright (c) 2023-2024 FlyByWire Simulations
 // SPDX-License-Identifier: GPL-3.0
 
-import { FeatureType, PolygonalStructureType } from '@flybywiresim/fbw-sdk';
+import { AmdbProperties, FeatureType, PolygonalStructureType } from '@flybywiresim/fbw-sdk';
+
+export interface LayerSpecification {
+  /** The scale at which to render this layer, as a multiple of the total size (in projected coordinates) of the entire airport map */
+  renderScale: number;
+
+  /** The visibility of this layer at each OANS zoom level */
+  zoomLevelVisibilities: [boolean, boolean, boolean, boolean, boolean];
+
+  /** The styling rules for this layer  */
+  styleRules: StyleRule[];
+}
 
 export interface StyleRule {
+  /** An array of feature types that this rule applies to */
   forFeatureTypes?: FeatureType[];
 
+  /** If {@link forFeatureTypes} is {@link FeatureType.VerticalPolygonalStructure}, an array that filters out which features to draw */
   forPolygonStructureTypes?: PolygonalStructureType[];
 
+  /** If `true`, the feature won't be fetched from the AMDB */
   dontFetchFromAmdb?: boolean;
 
-  styles: {
-    doStroke: boolean;
+  /** If applicable, how to union features this rule applies to. Only valid for polygons. */
+  unionBy?: keyof AmdbProperties | 'all';
+
+  /** If provided, the tolerance with which to simplify features. If not provided, the feature is not simplified. */
+  simplify?: number;
```

Comment:
Does this have units? Would be good to know how to choose a value from the JSDoc.